src/chainstate/burn/db/sortdb.rs

```diff
@@ -2213,15 +2184,10 @@ impl <'a> SortitionHandleTx <'a> {
             if let BlockstackOperationType::LeaderKeyRegister(ref data) = block_op {
                 keys.push(db_keys::vrf_key_status(&data.public_key));
                 values.push("1".to_string());       // indicates "available"
+                // TODO(psq): now that keys can be reused, does it matter what the value is?  As long as the key (of the key) exists
```

Comment:
The value shouldn't matter (but check the code first!), but the key/value pair must exist.  Let's leave it as "1" for now, just in case there are some code paths elsewhere that depend on it :)

Reply:
great, thank you for confirming my suspicion @jcnelson